packages/vite/src/node/logger.ts

```diff
@@ -77,7 +82,7 @@ export function createLogger(
               : type === 'warn'
               ? colors.yellow(colors.bold(prefix))
               : colors.red(colors.bold(prefix))
-          return `${colors.dim(new Date().toLocaleTimeString())} ${tag} ${msg}`
+          return `${colors.dim(timeFormatter.format(new Date()))} ${tag} ${msg}`
```

Comment:
It seems that `new Date()` can be omitted. 🤔

Reply:
It looks like that is the case, docs are not clear on MDN https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat/format. Maybe better to leave it explicit?

Reply:
Yes, I didn't find the documentation clearly stating whether it is optional or not. Only the ts hint is optional, as you said, it should be better to keep it as it is.